ada_hardware/launch/ada_hardware.launch.py

```diff
@@ -104,7 +109,7 @@ def generate_launch_description():
     start_rviz = LaunchConfiguration("start_rviz")
 
     use_sim_time = True
-    if sim == 'none':
+    if sim == "none":
```

Comment:
Shouldn't be addressed in this PR, but FWIW this line doesn't really make sense when we say that `sim` can take on the values: mock, real, or isaac.